force-app/main/default/lwc/serviceTypeIcon/__tests__/serviceTypeIcon.test.js

```diff
@@ -0,0 +1,71 @@
+import { createElement } from "lwc";
+import ServiceTypeIcon from "c/serviceTypeIcon";
+
+jest.mock("c/customIcon");
+describe("c-service-type-icon", () => {
+  const flushPromises = () => new Promise(process.nextTick);
+
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it("Test if showCostCalculator is true", async () => {
+    const element = createElement("c-service-type-icon", {
```

Comment:
where are the ARRANGE, ACT, ASSERT comments if you are not going to include them you better name your "it stamente" better otherwise one need to guess what this is testing, that is not ideal